src/utils/months.js

```diff
@@ -0,0 +1,57 @@
+import {getCurrentYear} from "./years";
+const currentMonth = (new Date()).getMonth()
+const months = [
```

Comment:
I may be missing something but why is this necessary?


![image](https://user-images.githubusercontent.com/3466388/50775924-d5edbd80-1297-11e9-9aab-262f9116e81f.png)

If you really need this mapping written by hand, it may be easier to use the built-in functions above, which will include locale as well.

Reply:
Googling "months and abbreviation json" was faster than writing code. 🤷‍♂️ 

This whole functionality will most likely be removed when API will start sending months as numbers (@dotadota counting on you!).

Reply:
Changed some stuff, so now your idea is used in the code. 😘